Makefile

```diff
@@ -32,6 +32,9 @@ package.zip: node_modules/ *.js package.json
 publish:
 	npm run rel
 
+update-collector-versions:
+	npm run bump-collector-versions $(VERSION)
```

Comment:
so this `$(VERSION)` is a paws-collector library version, right?

Reply:
If so can we automatically get the latest version from npm or `paws-collector/package.json` and update it here.

Reply:
We could. But i thought that would limit the utility of the script. I figured that we might want to roll a paws version back as well.

I could get the paws version from the latest package.json and use it in the script.

Reply:
> I figured that we might want to roll a paws version back as well

+1

> I could get the paws version from the latest package.json and use it in the script.

Let's have this as default behavior if `$VERSION` is not specified.

Reply:
Cool. Beefed up the script. It takes the latest version from the main package.json if the argument is omitted. and it doesnt bump the collector version if it is already up to date.